src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -0,0 +1,157 @@
+    canCoderConfig.MagnetSensor.SensorDirection = SensorDirectionValue.CounterClockwise_Positive;
+    canCoderConfig.MagnetSensor.MagnetOffset = 0.4;
+    canCoder.getConfigurator().apply(canCoderConfig);
+  }
+
+  private final VelocityTorqueCurrentFOC IntakeRollersMotorVelocity = new VelocityTorqueCurrentFOC(0, 0, 0, 1, false,
+      false,
+      false);
+
+  private final VelocityTorqueCurrentFOC IntakeDeployMotorVelocity = new VelocityTorqueCurrentFOC(0, 0, 0, 1, false,
+      false,
+      false);
+
+  /**
+   * Example command factory method.
+   *
+   * @return a command
+   */
+  public void intakeRollers(Measure<Voltage> volts) {
+    RollersMotor.setControl(voltageRequest.withOutput(volts.in(Volts)));
+  }
+
+  public void deploy(Measure<Voltage> volts) {
+    var intakeDeployConfig = new TalonFXConfiguration();
+    intakeDeployConfig.Slot0 = Slot0Configs.from(DEPLOY_SLOT_CONFIGS);
+    intakeDeployConfig.MotorOutput.Inverted = InvertedValue.CounterClockwise_Positive;
+    deployMotor.getConfigurator().apply(intakeDeployConfig);
+    deployMotor.setControl(voltageRequest.withOutput(volts.in(Volts)));
+  }
+
+  public void retractIntake(Measure<Voltage> volts) {
+    var intakeDeployConfig = new TalonFXConfiguration();
+    intakeDeployConfig.Slot0 = Slot0Configs.from(DEPLOY_SLOT_CONFIGS);
+    intakeDeployConfig.MotorOutput.Inverted = InvertedValue.Clockwise_Positive;
+    deployMotor.getConfigurator().apply(intakeDeployConfig);
+    deployMotor.setControl(voltageRequest.withOutput(volts.in(Volts)));
+  }
+
+  // Spit means to reverse the motor direction to 'spit' the game piece out
+  /**
+   * @return
+   */
+  public void spit(double rps) {
+    deployMotor.setControl(IntakeRollersMotorVelocity.withVelocity(0 - rps));
+  }
+
+  public Command sysIdDeployMotorQuasiCommand(Direction direction) {
+    return deployMotorSysIdRoutine.quasistatic(direction).withName("SysId Deploy Motor Quasistatic " + direction)
+        .finallyDo(this::stop);
+  }
+
+  public Command sysIdDeployMotorDynamCommand(Direction direction) {
+    return deployMotorSysIdRoutine.dynamic(direction).withName("SysId Deploy Motor Quasistatic " + direction)
+        .finallyDo(this::stop);
+  }
+
+  public Command sysIdRollersMotorQuasiCommand(Direction direction) {
+    return rollersMotorSysIdRoutine.quasistatic(direction).withName("SysId Rollers Motor Quasistatic " + direction)
+        .finallyDo(this::stop);
+  }
+
+  public Command sysIdRollersMotorDynamCommand(Direction direction) {
+    return rollersMotorSysIdRoutine.dynamic(direction).withName("SysId Rollers Motor Quasistatic " + direction)
+        .finallyDo(this::stop);
+  }
+
+  public void stop() {
+    // READ THIS!!!!!
+    // IDK what the motor velocity number should actually be, just set it to 0 for
+    // now
+    deployMotor.setControl(IntakeDeployMotorVelocity.withVelocity(0));
+    RollersMotor.setControl(IntakeRollersMotorVelocity.withVelocity(0));
```

Comment:
Don't set the velocity to zero to stop the motors. This will actively apply opposing voltage to the motor to stop it and hold it at a velocity of zero. We just want to apply idle output, so use `stopMotor()`.